### PR TITLE
Lu 3732 wrangler generic

### PR DESCRIPTION
**Pull request by ranjit-das**

### CODE REVIEW

File: handler.go [hunk 1]
```diff
@@ -20,10 +34,107 @@ func HandleLambdaEvent(config Config) (ValidationOutputWrapper, error) {
...
+		fmt.Println(err)
+	}
+	fmt.Printf("[Send message] \n%v \n\n", sendResp)
 
 }
 
 func main() {
-	lambda.Start(HandleLambdaEvent)
+
+	lambda.Start(handler)
```
Comment (RyanKilbane) [inline handler.go]: If this code is commented out, can we remove it?

Comment (ranjit-das) [reply handler.go]: Commented codes are removed.

File: serverless.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+functions:
+  main:
+    handler: bin/main
+    role: ${file(./config.dev.json):ROLE}
+    # layers:
+    #   - ${file(./config.dev.json):LAYER_NAME}
+    environment:
+      INPUT_QUEUE_URL: ${file(./config.dev.json):INPUT_QUEUE_URL}
+      OUTPUT_QUEUE_URL: ${file(./config.dev.json):OUTPUT_QUEUE_URL}
+    # vpc:
```
[the remark this answers is not in the document]
Comment (ranjit-das) [reply serverless.yml]: Not required to put inside vpc as it is not accessing any database. Also without vpc, execution time will be less.

**Comment by dllmorgan** [comment]
Could we please rename response_offset to period_offset

**Pull request merged**